llm/backfill_summaries.py

```diff
@@ -0,0 +1,67 @@
+# This script fills any missing 'summary' or 'topics' fields on the data model.
+# The document must have a 'Title' and 'DocumentText' field to generate them.
+#
+# Developer notes:
+# - you'll need to set the 'OPENAI_API_KEY' environment variable
+import firebase_admin
+from llm_functions import get_summary_api_function, get_tags_api_function_v2
+from firebase_admin import firestore
+from bill_on_document_created import get_categories_from_topics, CATEGORY_BY_TOPIC
+
+# Application Default credentials are automatically created.
+app = firebase_admin.initialize_app()
+db = firestore.client()
+
+
+# Conceptually, we want to return a very consistent format when generated status reports.
+# It would allow us to skip LLM regeneration when moving from dev to production.
+def make_bill_summary(bill_id, status, summary, topics):
+    return f"{bill_id},{status},{summary},{topics}\n"
+
+
+bills_ref = db.collection("generalCourts/194/bills")
+bills = bills_ref.get()
+# TODO: I need to use csv.writer here to properly escape quotes
```

Comment:
Using `f.write` below is 🎃. We should really use something that is designed to handle CSV i.e. `csv.writer` which would properly escape quotes, newlines, etc.

I did not originally know all the possibilities around summary formats.